infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -0,0 +1,48 @@
+# Initialize and Run Azure Sandbox
+Ensure you have a Azure account. As part of sandbox we use Terraform to provision a resource group, storage account, and blob container within that storage account. 
+
+## Azure Login
+
+Install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/), and ensure that you are logged in with your Azure account. You can login in multiple ways.
+
+### Using username/password
+
+This is the simplest version. Run `az login` and login to your account on a web browser.
+
+### Using Service Principal
+
+Following the steps [here](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-service-principal), you can run `az login --service-principal -u <app-id> -p <password-or-cert> --tenant <tenant>` to gain more control over your subscription and account.
+
+## Terraform Backend
+ 
+Terraform backend state can be stored locally or in Azure blob store. To configure this you will be setting the variables like those in `infra/recipes/terraform/azure/backend.tfvars.template`.
+
+### Using Environment Variables
+You could set the environment variables for variables to stored terraform backend state. For example, to initialize `storage_account_name` you can set the following environment variable.
+`export TF_VAR_storage_account_name="foobar"`
+
+### Using tfvars
+1. Create a copy of the file backend.tfvars.template by running
+`cp backend.tfvars.template backend.tfvars`.
+2. Edit the backend.tfvars to configure the necessary TF backend state.
+
+## Deployment
+
+To deploy the sandbox, run:
+
+### Initialization
+
+Run `terraform init` when setting up a new configuration, changing backend settings, or modifying provider versions. You don't need to run it for regular operations like `terraform plan` and `terraform apply` unless the configuration changes.
+
+_If using tfvars_
+`terraform init -backend-config="./backend.tfvars"`
+_If using environment variables_
+`terraform init`
+
+### Planning & Applying
+
+`terraform plan` to see the plan generated by terraform for the specified deployment configuration. If you are satisfied with the plan, run `terraform apply` to apply the configuration.
+
+## Deconfiguration
+
```

Comment:
Can we add a doc on module as there are 4 modules? For example, about directory structure, existing modules and adding a new module, etc places to update like main.tf etc.

Reply:
Yes. From my understanding this means one overall doc? I will do that, but let me know if you mean a doc per module instead?

Reply:
Either way it is fine. We can have a single doc and link the the existing README or in the exiting README itself we can add doc for each module and how to add new module.

Reply:
I guess overall general doc would be better capture the directory structure and other things like providers, top level variables and module specific variables, versions etc.